### ISSUE TITLE
Separate Mssql connection string, one for UseMssqlReadModel, the other for UseMssqlEventStore

### ISSUE THREAD

**Issue by HistoricallyCorrect** [eventflow/EventFlow#436]
This project looks awesome and I'm doing a PoC and hopefully will migrate some of the features to using this project where we need auditability, approval and change management abilities utilizing the natural power cqrs+es. 

I'm trying to use two databases one for read model (which I'm writing to existing tables), one for event store (want a separate db for separation of concerns). It doesn't seem to be possible to configure in **EventFlowOptions** to have two separate connection strings. Unless I separate the command / query into different apps, and connect the two with message queuing (thinking out loud here). But I guess I could write my own implementation for ReadStore which basically picks up a different connection string and the rest stays the same. If this is possible already, could someone point it out to me? Thanks. Otherwise I'll mark this as an enhancement. 

```
using (var resolver = EventFlowOptions.New.AddEvents(typeof(MyEvent))
.AddCommands(typeof(MyCommand))
.AddCommandHandlers(typeof(MyCommandHandler))

//maybe a dedicated method for registering ReadModelLocator
.RegisterServices(register => register.RegisterType(typeof(MyReadModelLocator)))

//need to be able to configure connection string for ReadModelOnly
.UseMssqlReadModel<MyReadModel, MyReadModelLocator>()
.ConfigureMsSql(MsSqlConfiguration.New.SetConnectionString(@"Server=.;Database=ExistingDb;Integrated Security=true"))

//need to be able to configure connection string for ReadModelOnly
.UseMssqlEventStore()
.CreateResolver())
```

**Comment by rasmus** [comment]
It isn't currently possible. As you said, you'll have to do some IoC fiddling, i.e., either decorate the implementation or inherit from the current MSSQL read model store implementation and use a different interface that inherits from `IMsSqlConnection` and pass that along.

Its been requested before and makes sense to be able to configure it.

**Comment by HistoricallyCorrect** [comment]
Thanks for the reply and suggestions.

**Comment by rasmus** [comment]
The is implemented and released in 1.0.4748-alpha

Have a look at https://github.com/eventflow/EventFlow/blob/develop-v1/MIGRATION_GUIDE.md#notable-new-features-in-1x
